packages/runtime-vapor/src/component.ts

```diff
@@ -2,6 +2,7 @@ import { type Ref, EffectScope, ref } from '@vue/reactivity'
 import type { Block } from './render'
 import type { DirectiveBinding } from './directive'
 import type { Data } from '@vue/shared'
+import { LifecycleHooks } from './apiLifecycle'
```

Comment:
```suggestion
import type { LifecycleHooks } from './apiLifecycle'
```

Reply:
sorry bro, LifecycleHooks is enum, i cat't use type keyword 👀